src/background/index.ts

```diff
@@ -1,15 +1,15 @@
-import { browser } from "webextension-polyfill-ts";
+import { latestReleaseUrl } from "utils/config";
+import { getStorageValue, setIcon, setStorageValue } from "utils/helpers";
 
 const getFirstRun = async () => {
-  const { abfFirstRun } = await browser.storage.local.get("abfFirstRun");
+  const { abfFirstRun } = await getStorageValue("abfFirstRun");
   return abfFirstRun as boolean;
```

Comment:
@Artini04 i have a nice change in #28 which will allow type inference when using `getStorageValue`, so using `as` like this won't be necessary after that goes in. leaving this here for now though